src/gmp/commands/credentials.js

```diff
@@ -133,8 +135,26 @@ class CredentialsCommand extends EntitiesCommand {
   }
 
   getEntitiesResponse(root) {
-    return root.get_credentials.get_credentials_response;
+    return normalizeCredentialKdcsArray(
+      root.get_credentials.get_credentials_response,
+    );
+  }
+}
+
+function normalizeCredentialKdcsArray(credentialResponse) {
+  if (Array.isArray(credentialResponse.credential)) {
+    credentialResponse.credential = credentialResponse.credential.map(cred => {
+      if (cred.kdcs && cred.kdcs.kdc) {
+        cred.kdcs = Array.isArray(cred.kdcs.kdc)
+          ? cred.kdcs.kdc
+          : [cred.kdcs.kdc];
+      } else {
+        cred.kdcs = [];
+      }
+      return cred;
+    });
   }
+  return credentialResponse;
 }
```

Comment:
This function doesn't belong the the command. It should be part of the Credential model parsing instead.